cmd/kube-controller-manager/app/options/options.go

```diff
@@ -75,8 +75,10 @@ func NewCMServer() *CMServer {
 			ConfigureCloudRoutes:              true,
 			TerminatedPodGCThreshold:          12500,
 			VolumeConfiguration: componentconfig.VolumeConfiguration{
-				EnableHostPathProvisioning: false,
-				EnableDynamicProvisioning:  true,
+				EnableHostPathProvisioning:       false,
+				EnableDynamicProvisioning:        true,
+				EnableNetworkStorageProvisioning: true,
+				StorageConfigDir:                 "/tmp",
```

Comment:
dtto, no Gluster/Ceph configuration outside of StorageClass.Parameters. It's a free-form `map[string]string`, you can squeeze anything you want there.